e2e/tests/omnibus-mainnet.toml/BFP_Market_ETH.e2e.js

```diff
@@ -0,0 +1,424 @@
+    log({ oldDepositedWeth });
+    log(collateralAddress);
+
+    assert.equal(
+      parseFloat(ethers.utils.formatEther(oldDepositedWeth.available)),
+      0,
+      'New account has 0 deposited WETH'
+    );
+
+    await contractWrite({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'modifyCollateral',
+      args: [accountId, marketId, collateralAddress, ethers.utils.parseEther(String(500))],
+    });
+
+    const newDigest = await contractRead({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'getAccountDigest',
+      args: [accountId, marketId],
+    });
+    log({ newDigest });
+
+    const newDepositedWeth = newDigest.depositedCollaterals.find(
+      (c) => c.collateralAddress === collateralAddress
+    );
+    log({ newDepositedWeth });
+
+    assert.equal(parseFloat(ethers.utils.formatEther(newDepositedWeth.available)), 500);
+  });
+
+  it('should open a short', async () => {
+    const marketId = require('../../deployments/extras.json').eth_market_id;
+    const currentPosition = await getBfpPosition({ accountId, marketId });
+    const collateralAddress = require('../../deployments/extras.json').weth_address;
+
+    assert.equal(currentPosition.positionSize, 0);
+
+    const newDigest = await contractRead({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'getAccountDigest',
+      args: [accountId, marketId],
+    });
+    log({ newDigest });
+
+    const newDepositedWeth = newDigest.depositedCollaterals.find(
+      (c) => c.collateralAddress === collateralAddress
+    );
+    log({ newDepositedWeth });
+    const { now, blockTimestamp } = getTimes(provider);
+    const buffer = 1; // 1s
+    const diff = now - blockTimestamp;
+
+    if (diff < 0) {
+      await wait(Math.abs(diff) + buffer * 1000);
+    }
+    await commitBfpOrder({
+      wallet,
+      accountId,
+      marketId,
+      sizeDelta: -0.01,
+    });
+
+    await wait(10000);
+    const newPosition = await settleBfpOrder({ wallet, accountId, marketId });
+
+    assert.equal(newPosition.positionSize, -0.01);
+  });
+
+  it('should close the short', async () => {
+    const marketId = require('../../deployments/extras.json').eth_market_id;
+    const currentPosition = await getBfpPosition({ accountId, marketId });
+
+    assert.equal(currentPosition.positionSize, -0.01);
+    const { now, blockTimestamp } = getTimes(provider);
+    const buffer = 1; // 1s
+    const diff = now - blockTimestamp;
+
+    if (diff < 0) {
+      await wait(Math.abs(diff) + buffer * 1000);
+    }
+    await commitBfpOrder({
+      wallet,
+      accountId,
+      marketId,
+      sizeDelta: 0.01,
+    });
+
+    // Wait for commitment price/settlement delay
+    await wait(2000);
+
+    // Wait for pyth to update prices
+    await wait(5000);
+    const newPosition = await settleBfpOrder({ wallet, accountId, marketId });
+
+    assert.equal(newPosition.positionSize, 0);
+  });
+
+  it('should mint and withdraw 1000 sUSD', async () => {
```

Comment:
I don't think we need any of these tests (borrow/withdraw) for CoreProxy. Why would we need them (given we have separate Liquidity_Provider test)?

Reply:
It's mostly to test calculation and payback of debt in the next test